.travis.yml

```diff
@@ -13,10 +15,11 @@ env:
   - DB=pgsql POSTGRESQL_VERSION=9.3
 
 before_script:
-  - sh -c "if [ '$DB' = 'pgsql' ]; then sudo /etc/init.d/postgresql stop; sudo /etc/init.d/postgresql start $POSTGRESQL_VERSION; fi"
```

Comment:
Are you sure that correct PostgreSQL version is used and POSTGRESQL_VERSION is used anywhere?